packages/theme-tools/test/color.test.ts

```diff
@@ -0,0 +1,30 @@
+import { getColor } from "../src"
+
+test("gets color from theme", () => {
+  expect(
+    getColor({ colors: { red: { 200: "#ff0000" } } }, "red.200", "red.200"),
+  ).toBe("#ff0000")
+})
+
+test("can get semantic token from theme", () => {
+  expect(
+    getColor(
+      { semanticTokens: { colors: { error: { 200: "#ff0000" } } } },
```

Comment:
The semanticTokens property of the theme only supports one level of nesting. E.g.

```js
{ semanticTokens: { colors: { "error.200": "#ff0000" } } }
```

And I think the lodash get function has does not quite work with this.